understanding/20/headings-and-labels.html

```diff
@@ -30,6 +30,9 @@ <h2>Intent of Headings and Labels</h2>
       <p>Labels and headings do not need to be lengthy. A word, or even a single character,
          may suffice if it provides an appropriate cue to finding and navigating content.
       </p>
+       <p>Labels of form controls are usually text-based. In some cases, images can serve as descriptive labels without additional text. In these cases, authors would need to ensure that the image and its use as label in a particular context can be considered conventional and widely understood. 
+         Note that the same image can be interpreted differently in different contexts. However, it can still be considered descriptive if its use is commonly understood in each context. For example, when accompanying a text field, a loupe or magnifying glass icon with text alternative of "Search" is commonly interpreted as indicating the field is for entering and submitting a search query. 
```

Comment:
```suggestion
         <p>Note that the same image can be interpreted differently in different contexts. However, it can still be considered descriptive if its use is commonly understood in each context. For example, when accompanying a text field, a loupe or magnifying glass icon with text alternative of "Search" is commonly interpreted as indicating the field is for entering and submitting a search query.</p>
```